sleep-for-days/sleepfordays_workflow.go

```diff
@@ -0,0 +1,37 @@
+package helloworld
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"go.temporal.io/sdk/activity"
+	"go.temporal.io/sdk/workflow"
+)
+
+const CompleteSignal = "complete"
+
+func SleepForDaysWorkflow(ctx workflow.Context, days int) (string, error) {
+	ctx = workflow.WithActivityOptions(ctx, workflow.ActivityOptions{
+		StartToCloseTimeout: 10 * time.Second,
+	})
+
+	isComplete := false
+	sigChan := workflow.GetSignalChannel(ctx, CompleteSignal)
+	workflow.Go(ctx, func(ctx workflow.Context) {
+		sigChan.Receive(ctx, &isComplete)
+	})
+
+	for !isComplete {
```

Comment:
Might be simpler to just check the signal channel every loop, also this sample is a bit different then the python one https://github.com/temporalio/samples-python/commit/2f3f2ac03d75e854e592b476354c0d692eb23f3b. In the python one your signal takes no input, but the Go one does. I think we should match python here.

Reply:
Removed the signal input (and workflow input).